src/main/target/CRAZYBEEF3FS/target.mk

```diff
@@ -0,0 +1,8 @@
+F3_TARGETS  += $(TARGET)
+FEATURES  = VCP 
+TARGET_SRC = \
+            drivers/accgyro/accgyro_mpu.c \
+            drivers/accgyro/accgyro_spi_mpu6000.c \
+			drivers/rx/rx_a7105.c \
```

Comment:
This target seems to be identical to the CRAZYBEEF3FR except for the fact that a different RX is on board. This means that it will be better and easier to maintain if it is defined as a variant of the CRAZYBEEF3FR target. For an example on how to do this for the different required drivers see https://github.com/betaflight/betaflight/blob/5dc23e662fc9de719bed42781b2447844771d75a/src/main/target/SPRACINGF3/target.mk#L13-L25